app/Resources/assets/js/eventshow.js

```diff
@@ -88,6 +88,17 @@ eventmetrics.eventshow.setState = function (state) {
     } else {
```

Comment:
Not really a blocker or anything, but you can also use $( 'body' ).toggleClass( 'disabled-state', 'started' === state );

Reply:
I like it! I'll make this change. I need to rebase in master anyway.